app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -216,7 +216,7 @@
     padding: 0
 
     .select2-highlighted
-      background: $header-item-bg-hover-color !important
+      background: $primary-color !important
```

Comment:
This effectively changes the color from `primary-color-dark` to `primary-color`.

Just noting that, I think it still looks good and is consistent, thus I will merge the PR anyway.